crates/sui-graphql-rpc/src/main.rs

```diff
@@ -38,6 +39,11 @@ async fn main() {
             println!("Starting server...");
             start_example_server(conn, service_config).await;
         }
+        Command::FromConfig { path } => {
+            let server = Server::from_yaml_config(path.to_str().unwrap());
```

Comment:
Mildly vexing that we are using both TOML and YAML to configure the same project -- can we just use one of them? I don't really mind which, but I tend to pick TOML because that's what's around us for Cargo, Move, etc.